portfolio-jsm/src/app/casestudies/morrent/page.tsx

```diff
@@ -1,11 +1,29 @@
+import ProjectDesc from "@/src/components/ProjectDesc";
+import ProjectRole from "@/src/components/ProjectRole";
+import ProjectTitle from "@/src/components/ProjectTitle";
 import React from "react";
 
-const page = () => {
+export default function Morrent() {
   return (
-    <main className="flex min-h-screen flex-col items-center text-center dark:text-white-800 md:p-24">
-      MORRENT
-    </main>
-  );
-};
+    <>
+      <ProjectTitle
+        title="Morrent"
+        desc="A Car Rental Website"
+        lImage="./job-image/Morrent-Laptop.svg"
+        mImage="./job-image/Morrent-Mobile.svg"
```

Comment:
```suggestion
        lImage="/job-image/Morrent-Laptop.svg"
        mImage="/job-image/Morrent-Mobile.svg"
```

Don't use a relative path here, that's why the image isn't loading.